src/python/org_fetcher/fetch_orgs.py

```diff
@@ -0,0 +1,64 @@
+from collections import defaultdict
+import csv
+import os
+
+PATH_TO_DATAFILES = os.path.join(
+    '..', '..', 'php', 'gov-harvester', 'datafiles'
+)
+
+
+orgs = defaultdict(lambda: 0)
+for filename in os.listdir(PATH_TO_DATAFILES):
+    filename_lower = filename.lower()
+    if not filename_lower.endswith('.csv'):
+        # Skip the file if it's not a csv
+        continue
+    if not 'meetings' in filename_lower:
+        # Skip the file if it doesn't contain a meeting of some sort
+        continue
+    if 'gift' in filename_lower:
+        # Skip the file if it also contains gift information
+        continue
+    with open(os.path.join(PATH_TO_DATAFILES, filename), 'rU') as file_handle:
+        reader = csv.reader(file_handle)
+        first_row = True
+        org_col = None
+        for row in reader:
+            if len(row) < 3:
+                # If there's fewer than three columns this row is unlikely to be useful
+                # Mostly we'd want 4 but sometimes it's meetings for XXX followed by a
+                # three column version with the other details.
+                continue
+            if first_row and not row[1]:
```

Comment:
I'd propose a similar pattern as above with `should_skip`. The comments can then be either removed or moved down to that function, to keep the main algorithm neat and tidy 😺